tests/draft2019-09/maxContains.json

```diff
@@ -0,0 +1,49 @@
+[
+    {
+        "description": "maxContains without contains is ignored",
+        "schema": {
+            "maxContains": 1
+        },
+        "tests": [
+            {
+                "description": "valid when valid against lone maxContains",
+                "data": [ 1 ],
+                "valid": true
+            }
+        ]
```

Comment:
I’d add a passing test with `data: [ 1, 1 ]` to ensure it’s not erroneously applied.